src/components/PostAComment.jsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useContext } from "react"
+import { UserLoginContext } from "../contexts/User";
+import { postComment } from "../utils/api";
+import { NoLoginMsgForCommment, PostCommentFailMsg, PostCommentSuccessMsg } from "../utils/messages"
+
+function PostAComment ({article_id, setSubmitComments, submitComments}) {
+
+const { userLogin } = useContext(UserLoginContext)
+
+const [ submitStatus, setSubmitStatus ] = useState("pending")
+const [ myComment, setMyComment ] = useState("")
+
+const handleLogin = (event) => {
```

Comment:
Shoudn't this be called handleSubmit, rather than handleLogin? As it's dealing with what to do when a new comment is submitted.

Reply:
oh yes it is my bad.